docs/_posts/2018-02-22-units-of-measurement.md

```diff
@@ -0,0 +1,109 @@
+---
+layout: post
+title: "Units of Measurement with Eclipse SmartHome"
+date:   2018-02-22
+image: "2018-02-22-units-of-measurement.png"
+published: false
+---
+
+A lot of functionality in a smart home setup is based on environmental conditions which until now are measured without an explicit unit. Now Eclipse SmartHome comes with full "Units of Measurement" support to make the unit part of the item state and provide automatic or specific unit conversion. <!--more--> 
+
+##### Overview
+A lot of functionality in a smart home setup is based on environmental conditions. Sensors will report temperatures from all sorts of locations, other weather data is collected and displayed or in house luminance is measured to control steady lighting conditions.
+All these sensor data is not only measured with a scalar value which represents the current state but has an implicit unit attached to it: Temperatures are usually given in either centigrade or degree Fahrenheit, wind speed will be displayed in either kilometers per hour (km/h) or miles per hour (mph) and luminance is given in Lux or it may even be a device specific unit or value scale.
+Up to now these units where not represented in Eclipse SmartHome and the solution/user had to "know" its units and model items and rules accordingly.
+In order to unify the different values and units and let the smart home make use of it, Eclipse SmartHome now comes with support for "Units of Measurement" or in short "uom".
```

Comment:
short "uom". -> short "UoM".

Though, I am not 100% sure about that. But I think its a defined term.